### PR TITLE
Add 2018

### PR DESCRIPTION
**Pull request by m1guelpf**
TO DO:

- [x] Wait to the next year (100% done)
- [ ] Merge!

:octocat:

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately we couldn't find an account corresponding to your GitHub username on [bugs.python.org](https://bugs.python.org/) (b.p.o) to verify you have signed the CLA (this might be simply due to a missing "GitHub Name" entry in your b.p.o account settings). This is necessary for legal reasons before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.


Thanks again to your contribution and we look forward to looking at it!

**Comment by duttaditya18** [comment]
@m1guelpf Signing the PSF Contributor Agreement is required for contributing to Python.

**Comment by m1guelpf** [comment]
@the-knights-who-say-ni @duttaditya18 Signed!

File: LICENSE [hunk 1]
```diff
@@ -73,7 +73,7 @@ analyze, test, perform and/or display publicly, prepare derivative works,
 distribute, and otherwise use Python alone or in any derivative version,
 provided, however, that PSF's License Agreement and PSF's notice of copyright,
 i.e., "Copyright (c) 2001, 2002, 2003, 2004, 2005, 2006, 2007, 2008, 2009, 2010,
-2011, 2012, 2013, 2014, 2015, 2016, 2017 Python Software Foundation; All Rights
+2011, 2012, 2013, 2014, 2015, 2016, 2017 2018 Python Software Foundation; All Rights
```
Comment (timgraham) [inline LICENSE]: Missing a comma after 2017 and it looks like "Rights" should flow to the next line to maintain consistent wrapping.

**Review by timgraham — COMMENTED** [review]
There are more places to update based on last year's commit: e527dd34bc4d894b469f48af9369b0f8e1138ea9.

File: LICENSE (hunk 1 above)
Comment (rooterkyberian) [inline LICENSE]: missing comma
this addition makes also this line only one that has more than 80 characters (newline characters excluded)

Not sure if this is fine in legal-speak, but chaning it to `2001-2018` would make this thing much less of a hassle to maintain and read.

**Review by rooterkyberian — CHANGES_REQUESTED** [review]

**Comment by duttaditya18** [comment]
@m1guelpf I would suggest you to squash the commits.

**Comment by asvetlov** [comment]
@duttaditya18 Pull Request is squashed on merging anyway.

**Comment by duttaditya18** [comment]
@asvetlov Oh yeah! Forgot that!

**Comment by m1guelpf** [comment]
@duttaditya18 Done!

**Comment by duttaditya18** [comment]
@m1guelpf Awesome!!

**Comment by duttaditya18** [comment]
@m1guelpf Awesome! Now wait for the merger!

**Comment by asvetlov** [comment]
Done by @benjaminp in #5094 #5104 #5105

**Comment by duttaditya18** [comment]
@m1guelpf Dang dude! Couldn't get your contribution to Python. Keep trying though.

**Pull request closed without merging**